components/bl-leafletMap-component/component.json

```diff
@@ -0,0 +1,320 @@
+      "handlerId": "mapVisibilityHandler",
+      "handlerLabel": "Map Visibility Logic",
+      "handlerDescription": "Used to control component visibility.",
+      "dataBinding": true
+    },
+    {
+      "type": "text",
+      "name": "center",
+      "label": "Center",
+      "defaultValue": "40.74, -73.96",
+      "handlerId": "centerHandler",
+      "handlerLabel": "Center Logic",
+      "dataBinding": true,
+      "handlerDescription": "Used to set center on map."
+    },
+    {
+      "type": "number",
+      "name": "zoom",
+      "label": "Zoom",
+      "defaultValue": "10",
+      "handlerId": "zoomHandler",
+      "handlerLabel": "Zoom Logic",
+      "dataBinding": true,
+      "handlerDescription": "Used to set zoom on map."
+    },
+    {
+      "type": "text",
+      "name": "markers",
+      "handlerId": "markersHandler",
+      "handlerDescription": "Specifies an array of markers to display on the map",
+      "dataBinding": true,
+      "handlerLabel": "Markers Logic"
+    },
+    {
+      "name": "circles",
+      "handlerId": "circlesHandler",
+      "dataBinding": true,
+      "handlerDescription": "Specifies an array of circles to display on the map",
+      "handlerLabel": "Circles Logic",
+      "type": "text"
+    },
+    {
+      "name": "polygons",
+      "handlerId": "polygonsHandler",
+      "handlerDescription": "Specifies an array of polygons to display on the map",
+      "handlerLabel": "Polygons Logic",
+      "dataBinding": true,
+      "type": "text"
+    },
+    {
+      "type": "select",
+      "name": "mapType",
+      "label": "Map type",
+      "defaultValue": "openStreet",
+      "options": [
+        {
+          "value": "carto",
+          "label": "Carto"
+        },
+        {
+          "value": "cartoDark",
+          "label": "Carto Dark"
+        },
+        {
+          "value": "openStreet",
+          "label": "Open Street"
+        },
+        {
+          "value": "cyclosm",
+          "label": "CyclOSM"
+        },
+        {
+          "value": "esriWorldStreet",
+          "label": "Esri World Street"
+        }
+      ],
+      "handlerId": "mapTypeHandler",
+      "handlerLabel": "Map Type Logic",
+      "dataBinding": true,
+      "handlerDescription": "Used to set map type. Possible: carto, cartoDark, openStreet, cyclosm, esriWorldStreet"
+    },
+    {
+      "type": "checkbox",
+      "name": "zoomControl",
+      "label": "Zoom Control",
+      "defaultValue": true,
+      "handlerId": "zoomControlHandler",
+      "handlerLabel": "Zoom Control Logic",
+      "handlerDescription": "Specifies whether the zoom is allowed.",
+      "dataBinding": true
+    },
+    {
+      "type": "checkbox",
+      "name": "draggingControl",
+      "label": "Dragging Control",
+      "defaultValue": true,
+      "handlerId": "draggingControlHandler",
+      "handlerLabel": "Dragging Control Logic",
+      "handlerDescription": "Specifies whether the dragging is allowed.",
```

Comment:
This is a handler for the logic to determine if dragging is allowed.

And so on.